docs/tempo/website/troubleshooting/_index.md

```diff
@@ -0,0 +1,118 @@
+- If you still see nothing, please [file an issue](https://github.com/grafana/tempo/issues/new/choose) on the Tempo repository with a description and the configuration used.
+
+### Diagnosing and fixing issues with querying traces
+If you have determined that data has been ingested correctly into Tempo, then it is time to investigate possible issues with querying the data.
+
+The presence of the following errors in the Tempo Querier log files may explain why traces are missing:
+
+- `no org id`
+- `could not dial 10.X.X.X:3100 connection refused`
+- `tenant-id not found`
+
+Possible reasons for the above errors are:
+- Not connected to Tempo Querier correctly
+- Insufficient permissions
+
+
+#### Solution
+
+- Fixing connection issues
+  - In case the application is not connected to Tempo Querier correctly, update the `backend.yaml` configuration file so that it is attempting to connect to the right port of the querier.
+- Fixing insufficient permissions issue
+  - Verify that the Querier has the LIST and GET permissions on the bucket.
+
+
+## 2. Problem - Some of my traces are missing in Tempo
+
+This could happen because of a number of reasons and some have been detailed in this blog post -
+[Where did all my spans go? A guide to diagnosing dropped spans in Jaeger distributed tracing](https://grafana.com/blog/2020/07/09/where-did-all-my-spans-go-a-guide-to-diagnosing-dropped-spans-in-jaeger-distributed-tracing/).
+
+### Diagnosing the issue 
+
+If the pipeline is not reporting any dropped spans, check whether application spans are being dropped by Tempo. The following metrics help determine this -
+- `tempo_receiver_refused_spans`. The value of `tempo_receiver_refused_spans` should be 0.
+If the value of `tempo_receiver_refused_spans` is greater than 0, then the possible reason is the application spans are being dropped due to rate limiting.
+
+#### Solution
+- The rate limiting may be appropriate and does not need to be fixed. The metric simply explained the cause of the missing spans, and there is nothing more to be done.
+- If more ingestion volume is needed, increase the configuration for the rate limiting, by adding this CLI flag to Tempo at startup - https://github.com/grafana/tempo/blob/78f3554ca30bd5a4dec01629b8b7b2b0b2b489be/modules/overrides/limits.go#L42
+
+## 3. Problem: Getting error message ‘Too many jobs in the queue’
+You may see this error if the compactor isn’t running and the blocklist size has exploded. 
+Possible reasons why the compactor may not be running are:
+
+- Insufficient permissions.
+- Compactor sitting idle because no block is hashing to it.
+- Incorrect configuration settings.
+
+### Diagnosing the issue
+- Check metric `tempodb_compaction_bytes_written`
+If this is greater than zero (0), it means the compactor is running and writing to the backend.
+- Check metric `tempodb_compaction_errors_total`
+If this metric is greater than zero (0), it likely means there are permission related issues.
```

Comment:
i don't think we should say it's "likely permission issues".  It could be any of a large number of different issues so I think the guidance should be to check the logs and diagnose.

Including specific info about permissions here is good though.